stdlib/public/CTensorFlow/CMakeLists.txt

```diff
@@ -53,20 +56,31 @@ foreach(sdk ${TARGET_SDKS})
 
     # Copy headers.
     macro(copy_tf_headers src dst)
+      if (TF_PATH_ADJUSTMENT)
+        set(copy_tf_headers_path_adjustment "${TF_PATH_ADJUSTMENT}/")
+      else()
+        set(copy_tf_headers_path_adjustment "")
+      endif()
       add_custom_command(
         OUTPUT "${output_dir}/${dst}"
         DEPENDS "${src}"
         COMMAND
           "${SWIFT_SOURCE_DIR}/utils/fix-tf-headers.sh"
           "${src}"
-          "${output_dir}/${dst}")
+          "${output_dir}/${dst}"
+          "${copy_tf_headers_path_adjustment}")
 
       list(APPEND outputs "${output_dir}/${dst}")
     endmacro(copy_tf_headers)
 
-    copy_tf_headers("${TF_INCLUDE_DIR}/tensorflow/c/c_api.h" c_api.h)
-    copy_tf_headers("${TF_INCLUDE_DIR}/tensorflow/c/c_api_experimental.h" c_api_experimental.h)
-    copy_tf_headers("${TF_INCLUDE_DIR}/tensorflow/c/eager/c_api.h" c_api_eager.h)
+    if (TF_PATH_ADJUSTMENT)
+      set(adjusted_tf_include_dir "${TF_INCLUDE_DIR}/${TF_PATH_ADJUSTMENT}")
```

Comment:
should we make `adjusted_tf_include_dir` all capital, for consistency?

Reply:
If you notice above, the "local" sets are lowercase. I believe due to this I'm being consistent. Correct me if I'm wrong.